assets/content/cookbook/Beginner/arrays.md

```diff
@@ -77,152 +86,171 @@ That means these are your options:
 * If you iterate by increment the index you can get away with not increment the index it in case of removal (using `while` loop).
 * Or sometimes you might prefer to just build a new array instead.
 
-### Retrieving elements
+# Retrieving elements
 
 Array elements can be accessed through array notation, using the index of the element in the array.
 
 ```haxe
-var arrayOfStrings:Array<String> = ["first", "foo", "middle", "foo", "last"];
+var strings:Array<String> = ["first", "foo", "middle", "foo", "last"];
+
 // Retrieves first array element
-var first = arrayOfStrings[0];
+var first = strings[0];
+
 // Retrieves last array element
-var last = arrayOfStrings[arrayOfStrings.length - 1];
+var last = strings[strings.length - 1];
+
 // Retrieves first occurrence of "foo" string
-var firstOccurrence = arrayOfStrings[arrayOfStrings.indexOf("foo")];
+var first = strings[strings.indexOf("foo")];
+
 // Retrieves last occurrence of "foo" string
-var lastOccurrence = arrayOfStrings[arrayOfStrings.lastIndexOf("foo")];
+var last = strings[strings.lastIndexOf("foo")];
 ```
 
-### Iteration
+# Iteration
+
+The array defines an iterator, and its elements can therefore be iterated over in a for loop:
+
+```haxe
+var items = ["a", "b", "c"];
+for (item in items) {
+    trace(item);
+}
+// a
+// b
+// c
+```
 
-The array defines an iterator, and its elements can therefore be iterated over.
+Using the `index => item` notation in the for loop is the simplest way to get the current item as well as its index:
 
 ```haxe
-for (item in arrayOfStrings) {
-    // do something
+var var items = ["a", "b", "c"];
```

Comment:
`var var` :)

Reply:
Live code review - that's great! :-)
Fixed.